### PR TITLE
gcbmgr: add releases numbers tags to the cloudbuild

### PR DESCRIPTION
**Pull request by cpanato**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Add new tags to the cloud build to make a bit easier to search and filter in the future for analysis

the new tags that this PR adds are:
- `MAJOR_VERSION_TAG` will be something like: `1.18`
- `FULL_VERSION_TAG` will be something like: `1.18.0`
- `SUB_VERSION_TAG` will be something like: `beta.2`

#### Which issue(s) this PR fixes:
Fixes partially https://github.com/kubernetes/sig-release/issues/1005

#### Does this PR introduce a user-facing change?

```release-note
gcbmgr: add releases numbers tags to the cloudbuild
```

/assign @tpepper @saschagrunert

### CODE REVIEW

File: cmd/krel/cmd/gcbmgr.go [hunk 1]
```diff
@@ -347,6 +347,21 @@ func SetGCBSubstitutions(o *GcbmgrOptions, toolOrg, toolRepo, toolBranch string)
 	}
 	gcbSubs["KUBE_CROSS_VERSION"] = kubecrossVersion
 
+	// Adding more information in the cloud build bu adding some tags to describe the release version
+	// MAJOR_VERSION_TAG = 1.18
```
[the remark this answers is not in the document]
Comment (cpanato) [reply cmd/krel/cmd/gcbmgr.go]: thanks for your review. makes sense and made the changes :)

**Comment by cpanato** [comment]
@justaugustus thanks for your review! made the requested changes

**Comment by tpepper** [comment]
If splitting major, minor, patch, I suppose the -foo.N should also get sub splitting. It is not strictly part of th patch version

**Comment by cpanato** [comment]
@tpepper @justaugustus I like more having the previous approach plus the full version and having the `-foo.X` as a tag as well, so i propose the following

- `MAJOR_MINIOR_VERSION` -> `1.19`
- `MAJOR_VERSION` -> `1`
- `MINOR_VERSION` -> `19`
- `PATCH_VERSION` -> `0`
- `SUFIX_VERSION` -> `beta.4`
- `KUBERNETES_VERSION`  -> `1.18.4` or if is beta or other `1.19.0-beta.2`

**Pull request merged**